File: Selenium/Activities/Activity_13.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import java.util.List;

public class Activity_13 {
	public static void main(String[]args) {
		WebDriver driver=new FirefoxDriver();
		
	    driver.get("https://training-support.net/webelements/tables");
	    System.out.println("page_title"+driver.getTitle());
	  List<WebElement> column= driver.findElements(By.xpath("//table[contains(@class,'table-auto')]/thead/tr/th"));
	    System.out.println("no.of column:"+column.size());
	    List<WebElement> row= driver.findElements(By.xpath("//table[contains(@class,'table-auto')]/tbody/tr"));
	    System.out.println("no.of row:"+row.size());
	    //Find and print all the cell values in the third row of the table.
	    List <WebElement> row3=driver.findElements(By.xpath("//table[contains(@class,'table-auto')]/tbody/tr[3]/td"));
	    System.out.println("third row:");
	    for(WebElement cell:row3)
	       System.out.println(cell.getText());
	    //Find and print the cell value at the second row second column.
	    WebElement row_col= driver.findElement(By.xpath("//table[contains(@class,'table-auto')]/tbody/tr[2]/td[2]"));       
	   System.out.println("second row second column:"+ row_col.getText());
	    driver.quit();
}
}

File: Selenium/Activities/Activity-13.py
import selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/tables")
    print(driver.title)
    column= driver.find_elements(By.XPATH,"//table[contains(@class,'table-auto')]/thead/tr/th")
    print("no.of column:",len(column))
    row= driver.find_elements(By.XPATH,"//table[contains(@class,'table-auto')]/tbody/tr")
    print("no.of row:",len(row))
    #Find and print all the cell values in the third row of the table.
    row3=driver.find_elements(By.XPATH,"//table[contains(@class,'table-auto')]/tbody/tr[3]/td")
    print("third row:")
    for cell in row3:
        print(cell.text)
    #Find and print the cell value at the second row second column.
    row_colum=driver.find_element(By.XPATH,"//table[contains(@class,'table-auto')]/tbody/tr[2]/td[2]")       
    print("second row second column:",row_colum.text)
    driver.quit()


